usl_pipeline/usl_pipeline/formats/geotiff_loader.py

```diff
@@ -0,0 +1,39 @@
+import rasterio.features
+
+
+def load_elevation_from_geotiff(
+    file_path: str, load_data: bool = False, result_nodata: float = None
+):
+    """
+    Loading elevation raster data from GeoTIFF file.
+    :param file_path: path to a file to load from
+    :param load_data: indicates that in addition to header, data should also be
+            loaded
+    :param result_nodata: optional new value for special case of no data in
+            raster cells
+    :return: 3-size tuple (header, data, CRS of the raster)
+    """
+
+    with rasterio.open(file_path) as src:
+        transform = src.transform
+        ll_corner = transform * (0, src.height)
+        input_nodata = src.nodata
+        elv_header = {
+            "ncols": int(src.width),
```

Comment:
It looks like these are already ints and floats, so we shouldn't need to call the int & float functions here.

Reply:
Yeah, removed those (and switched to data object for header instead of dict).